.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -117,11 +117,6 @@ resources:
     env:
       DD_API_KEY: $(ddApiKey)
       DD_INSIDE_CI: true
-  repositories:
-  - repository: dd-continuous-profiler-dotnet
-    type: github
-    endpoint: DataDog
-    name: DataDog/dd-continuous-profiler-dotnet
 
 # Stages
 stages:
```

Comment:
i do not know what I should do starting at the line [188](https://github.com/DataDog/dd-trace-dotnet/pull/2404/files#diff-2e09bd52f55cbb3965ceeae7f556dbbd458b94a4566d9631bea1204a4ad12198R188)
@zacharycmontoya

Reply:
The last files of this PR are modifications related to building from the tracer repository instead of cloning the profiler repository

Reply:
For the steps in the `build_windows_profiler` stage:

You can delete the following steps
```
    - checkout: dd-continuous-profiler-dotnet
      path: dd-continuous-profiler-dotnet

    - powershell: |
        New-Item -Path $(System.DefaultWorkingDirectory)\..\dd-trace-dotnet -ItemType SymbolicLink -Value $(System.DefaultWorkingDirectory)
      displayName: Create symlink from $(System.DefaultWorkingDirectory)\..\dd-trace-dotnet to $(System.DefaultWorkingDirectory)

    - powershell: |
        Write-Host "Attempting to check out ref specified by GIT_PROFILER_REF: '$(GIT_PROFILER_REF)'"
        git checkout $(GIT_PROFILER_REF)
      workingDirectory: $(System.DefaultWorkingDirectory)\..\dd-continuous-profiler-dotnet
      displayName: Checkout GIT_PROFILER_REF
```

The `SetMsiSuffix` step is only useful if you want to include the commit SHA in the MSI version, like you were doing for nightly builds. If you don't care about that anymore, you can delete that step, but if you still want it I think you can get the SHA from the Build.SourceVersion variable instead, so the step would look like this:
```
    - powershell: |
        echo "##vso[task.setvariable variable=BetaMsiSuffix;isOutput=true]v$(Build.BuildId)-$(Build.SourceVersion)"
        Write-Host "BetaMsiSuffix set to v$(Build.BuildId)-$(Build.SourceVersion)"
      displayName: Set the commit sha for profiler build
      name: SetMsiSuffix # because we're going to depend on the variable output, we need to name the step
```

Reply:
thanks, I removed the steps.

Reply:
> if you still want it I think you can get the SHA from the Build.SourceVersion variable instead,

FYI, this doesn't _always_ work, but we define a variable `OriginalCommitId` which should work instead